keyboards/bajjak/bajjak.h

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+Copyright 2015 ZSA Technology Labs Inc (@zsa)
+Copyright 2020 Christopher Courtney <drashna@live.com> (@drashna)
+Copyright 2021 Gary Kong <kongkm88@gmail.com> (@garykong)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "quantum.h"
+#include <stdint.h>
+#include <stdbool.h>
+#include "i2c_master.h"
+
+// I2C aliases and register addresses (see "mcp23018.md")
+#define I2C_ADDR        0b0100000
+#define I2C_ADDR_WRITE  ( (I2C_ADDR<<1) | I2C_WRITE )
+#define I2C_ADDR_READ   ( (I2C_ADDR<<1) | I2C_READ  )
+#define IODIRA          0x00            // i/o direction register
+#define IODIRB          0x01
+#define GPPUA           0x0C            // GPIO pull-up resistor register
+#define GPPUB           0x0D
+#define GPIOA           0x12            // general purpose i/o port register (write modifies OLAT)
+#define GPIOB           0x13
+#define OLATA           0x14            // output latch register
+#define OLATB           0x15
+
+extern i2c_status_t mcp23018_status;
+#define BAJJAK_EZ_I2C_TIMEOUT 100
+
+void init_BAJJAK(void);
+void BAJJAK_blink_all_leds(void);
+uint8_t init_mcp23018(void);
+uint8_t BAJJAK_left_leds_update(void);
+
+#ifndef LED_BRIGHTNESS_LO
+#define LED_BRIGHTNESS_LO       15
+#endif
+#ifndef LED_BRIGHTNESS_HI
+#define LED_BRIGHTNESS_HI       255
+#endif
+
+
+inline void BAJJAK_board_led_on(void)      { DDRD |=  (1<<6); PORTD |=  (1<<6); }
```

Comment:
```c
inline void bajjak_board_led_on(void)      { DDRD |=  (1<<6); PORTD |=  (1<<6); }
```
etc.

Reply:
I don't understand.

Reply:
Change all the function names to lowercase.

Reply:
@fauxpark Changed function names to lowercase.